plugins/modules/lvg.py

```diff
@@ -72,6 +72,29 @@
     - If C(true), allows to remove volume group with logical volumes.
     type: bool
     default: false
+  old_vg:
+    description:
+    - The old name of the volume group.
+    - If specified, the volume group is renamed.
```

Comment:
I was thinking about it before implementation, and I'm thinking again. I intend to give a solution that makes it possible to create streamlined playbooks by avoiding too many tasks. Well, I admit, the current approach clashes with Unix philosophy.

Given that I have a cloned VG. I need to rename it and change the PV and VG UUIDs. 
With the current solution, this is only two tasks (later on the active/deactivate state :) )

```yaml

  - name: Ensure the clone VG is renamed and the PV and VG UUIDs are regenerated
    community.general.lvg:
      old_vg: "{{ source_vg_name }}"
      vg: "{{ clone_vg_name }}"
      pvs: "{{ clone_pv }}"
      active: false
      reset_vg_uuid: true
      reset_pv_uuid: true

  - name: Ensure the clone VG is active
    community.general.lvg:
      vg: "{{ clone_vg_name }}"
      active: true

```

With the proposed solution it would take three tasks:

1.  inactive
2.  renamed
3.  activate

What's your opinion on introducing a new `state=renamed`? We could require the `old_vg` and `vg` options for the new state, and set the `old_vg` mutually exclusive with other options.

Reply:
It might be better to add a new module instead, `lvg_rename`, instead of `state=renamed`. That simplifies the interface of the `lvg` module (and its implementation), and makes renaming also more simple by providing a streamlined module for it.

Reply:
I've made the necessary changes and created a new PR for the `lvg_rename` module: #6721